src/Kiota.Builder/Writers/CSharp/TypeDefinitionExtensions.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Text;
+using Kiota.Builder.CodeDOM;
+using Kiota.Builder.Extensions;
+
+namespace Kiota.Builder.Writers.CSharp;
+
+internal static class TypeDefinitionExtensions
+{
+    public static string GetFullName(this ITypeDefinition typeDefinition)
```

Comment:
Does it make sense to rename this to `GetFullyQualifiedName`?

Reply:
At first I thought the same, I think the commit history even includes a version I named it `GetFullyQualifiedName` or something.

The driving factors renaming it to `GetFullName` were two articles from the docs:
1. https://learn.microsoft.com/en-us/dotnet/fundamentals/reflection/specifying-fully-qualified-type-names
2. https://learn.microsoft.com/en-us/dotnet/api/system.type.fullname?view=net-8.0

> A fully qualified type name consists of an assembly name specification, a namespace specification, and a type name [1]

and

> Gets the fully qualified name of the type, including its namespace but not its assembly. [2]

Since there is no real chance to include an assembly name during generation, my thought was to go with the name of a specification of a types name without any assembly information - which would be `Type.FullName`

If you have any objections, please just feel free to share them, I'm not that good in naming things.

Reply:
Thanks for clarifying. This current naming works really given the reasoning here. Resolving.